src/LoginComponent.js

```diff
@@ -1,6 +1,10 @@
 import React from 'react';
 import firebase from 'firebase';
 
+
+// Why is this a class :(
+// I'm BIG MAD
+
 class LoginComponent extends React.Component {
```

Comment:
should this be a class?

Reply:
also probably gonna ask someone to explain whats going on here teehee

Reply:
implementing this as a class component is the old way in react, this can be converted to a functional component but tbh doesn't matter too much. If anything we can change it later after we have a better grasp of the project.

Reply:
Lool oops I made this before I saw your duke code